src/sql-parser/src/ast/defs/statement.rs

```diff
@@ -2882,6 +2882,7 @@ pub enum IfExistsBehavior {
 pub struct DeclareStatement<T: AstInfo> {
     pub name: Ident,
     pub stmt: Box<T::NestedStatement>,
+    pub sql: String,
```

Comment:
This is maybe unnecessary bike shedding, but is there a better name we could use to indicate that it's the SQL of the nested statement and not of the declare statement itself?